WalletWasabi/CoinJoin/Client/Clients/CoinJoinClient.cs

```diff
@@ -247,7 +248,7 @@ private async Task TryProcessStatusAsync(IEnumerable<RoundStateResponseBase> sta
 					}
 				}
 			}
-			StateUpdated?.Invoke(this, null);
+			StateUpdated?.Invoke(this, EventArgs.Empty);
```

Comment:
This is a breaking change so, make sure the event handlers do not take decisions based on this parameter value. I mean, if you change the value of an argument and the result is not affected the something is wrong somewhere.

Reply:
As far as I looked into it, StateUpdated is not subscribed to anywhere, it's probably a leftover, and as far as I know, it will be replaced.
So changing the null doesn't effect anything, it will just remove a warning for now, but didn't want to remove it yet, as it may be used later, until it get's removed completely.

Reply:
Yep, if it is dead code then it should be removed.

Reply:
There are some methods that have references throughout the project and tests.
I wish not to be the one to completely remove this (maybe the whole file/folder?) related to coinjoin, as I don't know which is neccessery/still used somewhere.

Reply:
I agree, can you remove the event?